KeyboardKit/KeyboardAction.swift

```diff
@@ -0,0 +1,136 @@
+    case zoomIn
+    
+    /// A zoom-out action, consisting of the minus (-) key and the Command (⌘) modifier.
+    case zoomOut
+    
+    /// An action to zoom content to its actual size, consisting of the 0 key and the Command (⌘) modifier.
+    case zoomToActualSize
+    
+    /// A rewind action, consisting of the left arrow (←) key and the Command (⌘) modifier.
+    case rewind
+    
+    /// A fast-forward action, consisting of the right arrow (→) key and the Command (⌘) modifier.
+    case fastForward
+}
+
+// MARK: - Key Equivalent
+
+extension KeyboardAction {
+    var keyEquivalent: (modifierFlags: UIKeyModifierFlags, input: String) {
+        switch self {
+        case .cancel:           return ([], .escape)
+        case .close:            return (.command, "w")
+        // Apparently "\u{3}" might work for enter (not return) but not quite. Shows up in the HUD with no key and I couldn’t get it to trigger. For now use cmd + return instead.
+        // Sources: https://forums.developer.apple.com/thread/119584 and https://stackoverflow.com/questions/56963348/uikeycommand-for-the-enter-key-on-mac-keyboards-numeric-keypad.
+        case .done:             return (.command, .returnOrEnter)
+        case .save:             return (.command, "s")
+        case .generic:          return (.command, "i") // Safari uses this for Email This Page. Also indirectly recommended in https://developer.apple.com/wwdc20/10117.
+        case .edit:             return (.command, "e")
+        case .new:              return (.command, "n")
+        case .reply:            return (.command, "r")
+        case .refresh:          return (.command, "r")
+        case .bookmarks:        return (.command, "b") // opt + cmd + B or shift + cmd + B might be better to be more like Safari.
+        case .search:           return (.command, "f")
+        case .delete:           return (.command, .delete)
+        case .today:            return (.command, "t")
+        case .zoomIn:           return (.command, "=")
+        case .zoomOut:          return (.command, "-")
+        case .zoomToActualSize: return (.command, "0")
+        case .rewind:           return (.command, .leftArrow)
+        case .fastForward:      return (.command, .rightArrow)
+        }
+    }
+}
+
+// MARK: - KeyboardShortcut
+
+#if canImport(SwiftUI)
+import SwiftUI
+
+@available(iOS 14.0, *)
+public extension KeyboardAction {
+    var keyboardShortcut: KeyboardShortcut {
+        let equivalent = keyEquivalent
+        let keyEquivalent = KeyEquivalent(Character(equivalent.input))
+        let modifiers = equivalent.modifierFlags.eventModifiers
+        
+        switch self {
+        case .cancel:
+            return .cancelAction
+        case .rewind, .fastForward:
+            if #available(iOS 15.0, *) {
+                return .init(keyEquivalent, modifiers: modifiers, localization: .withoutMirroring)
+            }
+            
+            fallthrough
```

Comment:
This feels a bit awkward. Other options: make the whole extension iOS 15.0+, or return nil for `.rewind` and `.fastForward` on iOS 14.0.